mlibrary/search
# Removing the need for `umich-lib-core-temp`.

# Overview
With the support of CSS `:root` variables, we no longer need `src/modules/reusable/umich-lib-core-temp/index.js`. Variables and utility classes have been created and applied to replace the JavaScript variables originally created and used.

## Anything else?
To align with [current React practices](https://react.dev/learn/your-first-component), these class components have been converted to functional components:
- `AdvancedSearchContainer `

## Testing
- Make sure the PR is consistent in these browsers:
- Run accessibility tests:
- Poke around the site and make sure everything still looks the same.
